windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -13,7 +13,7 @@
 #  ARGUMENTS
 ##############################
 
-Param([string]$version = "")
+Param([string]$version = (Get-Content "$PSScriptRoot\..\windows-installer\VERSION"))
 
 if ($version -eq "")
```

Comment:
[Optional] We might not need this test anymore. WDYT?

Reply:
Oh, line 18... Yes, my eyes were looking for where's the highlight..